### PR TITLE
Add AWS resource tag with the Giant Swarm Organization

### PR DESCRIPTION
**Pull request by rossf7**
Towards giantswarm/giantswarm#2695

### CODE REVIEW

File: service/awsconfig/v7/key/key.go [hunk 1]
```diff
@@ -108,11 +111,18 @@ func ClusterNamespace(customObject v1alpha1.AWSConfig) string {
 	return ClusterID(customObject)
 }
 
+// ClusterOrganization returns the org name from the custom object.
+// It uses ClusterCustomer until this field is renamed in the custom object.
+func ClusterOrganization(customObject v1alpha1.AWSConfig) string {
+	return ClusterCustomer(customObject)
+}
```
Comment (rossf7) [inline service/awsconfig/v7/key/key.go]: Adds a new key helper that just calls `ClusterCustomer`. Hopefully the comment is clear and we can fix when we clean up the `awsconfig` CRD.

File: service/awsconfig/v7/key/key.go [hunk 2]
```diff
@@ -108,11 +111,18 @@ func ClusterNamespace(customObject v1alpha1.AWSConfig) string {
...
+
 func ClusterTags(customObject v1alpha1.AWSConfig) map[string]string {
 	cloudProviderTag := ClusterCloudProviderTag(customObject)
 	tags := map[string]string{
-		cloudProviderTag: CloudProviderTagOwnedValue,
-		ClusterTagName:   ClusterID(customObject),
+		cloudProviderTag:    CloudProviderTagOwnedValue,
```
Comment (xh3b4sd) [inline service/awsconfig/v7/key/key.go]: Why is this lower case?

Comment (rossf7) [reply service/awsconfig/v7/key/key.go]: It's a variable. We need to call ClusterCloudProviderTag because the tag name includes the clusterID.

Comment (xh3b4sd) [reply service/awsconfig/v7/key/key.go]: Yeah, just wondering, the other vars are upper, but this is lower. #consistency

Comment (rossf7) [reply service/awsconfig/v7/key/key.go]: Cool good to check. The others are constants because their tag names are static.

**Pull request merged**